.github/workflows/lint-testdata-template.yml

```diff
@@ -0,0 +1,40 @@
+name: Testdata Lint Template
+
+# Trigger the workflow on workflow caller
+on:
+  # reusable workflow template
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+      working-directory:
+        required: true
+        type: string
+
+jobs:
+  check-testdata-lint:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Clone the code
+        uses: actions/checkout@v3
+      - name: Run linter for ${{ inputs.name }}
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.50
+          working-directory: ${{ inputs.working-directory }}
+          only-new-issues: true # Show only new issues if it's a pull request
```

Comment:
```suggestion
```

Reply:
I think is because of it we are not checking the issue right?

Reply:
However, I am OK with either we move in this way and then create a task for we sort out all issues as follow up.
What do you think ?